pkgs/development/python-modules/python-multipart/default.nix

```diff
@@ -50,8 +48,9 @@ buildPythonPackage rec {
   };
 
   meta = with lib; {
+    changelog = "https://github.com/Kludex/python-multipart/blob/${src.rev}/CHANGELOG.md";
```

Comment:
src.rev is probably wrong here

Reply:
Why would it?

Reply:
NVM, mixed it up with other recent tree wide changed